src/main/java/nl/knaw/dans/migration/core/DataverseLoader.java

```diff
@@ -106,6 +107,11 @@ public void loadFromDataset(String doi) {
             actualDataset.setDoi(shortDoi);
             actualDataset.setDepositor(depositor);
             actualDataset.setFileAccessRequest(v.isFileAccessRequest());
+            v.getMetadataBlocks()
+                .get("citation").getFields().stream()
+                .filter(field -> "dateOfDeposit".equals(field.getTypeName()))
```

Comment:
is this the date you are using as the citation date? The date i want is the publicationdate, as recorded in the dvobject table

Reply:
For now. 
Reading from the dvndb requires a lot of fuzz like an additional bundle. I prefer a separate PR for that purpose.